task/04-arrays-tasks.js

```diff
@@ -286,7 +341,10 @@ function propagateItemsByPositionIndex(arr) {
  *   [ 10, 10, 10, 10 ] => [ 10, 10, 10 ]
  */
 function get3TopItems(arr) {
-   throw new Error('Not implemented');
+   return arr.reverse().filter((elem, index) => {
+      if (index < 3)
+         return elem;
+  });
```

Comment:
Please, add sort method in solutin, because initial array may not be sorted (future cases)